tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -26,29 +19,42 @@ class TestUpgradeOCP(ManageTest):
     4. check all OCP ClusterOperators
     5. check OCP version
     5. monitor cluster health
+
     """
 
     def test_upgrade_ocp(self):
         """
         Tests OCS stability when upgrading OCP
 
+        How to creates config files for ocp upgrade:
```

Comment:
I am now much more confused from those informations.
We should have defined config files which allow us to do upgrade from GAed to nightly or GAed version.

User should use just ocp-4.4-upgrade.yaml or/ocp-4.4-ga-upgrade.yaml config file and don't care.

If they would like to upgrade from GAed version they should use https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocp_version/ocp-4.3-ga-minus1-config.yaml and then for upgrade ocp-4.3-ga-upgrade.yaml .

I see we are missing -1 config for 4.4 and 4.5, so we should have them and then upgrade our jenkins job to allow user to deploy previous version.